charts/argo-cd/values.yaml

```diff
@@ -2152,6 +2152,11 @@ repoServer:
   #  - name: cmp-tmp
   #    emptyDir: {}
 
+  # -- Toggle the usage of a shared Helm working directory
+  helmWorkingDir:
```

Comment:
This is map, not a feature toggle. Are we expecting another set of parameters that will be inside of map or is it just for enabling / disabling the directory? WDYT @mkilchhofer @jmeridth

Reply:
I think I originally had this setup as just a single boolean variable but thought it looked cleaner and like other values with and `enabled`. I also thought that perhaps making the path configurable in the future could be something that people might want so allowing it to expand could be helpful.

I can change the comment or make all of this into a single variable.

Reply:
The reason why I'm asking the rest of the team is that we had this historically in chart roughly 1 year ago for multiple features that were a map with single parameter and nothing else. These sections were eventually moved either to simple global toggles in top of the values.yaml or into cm and params sections if possible. Only one that I survived this cleanup is currently `openshift.enabled` until I can do some regression on OpenShift cluster.

Reply:
I think we should use a top-level `.Values.useEphemeralHelmWorkingDir` (or similar 😉) bool or move it below the `repoServer:` structure.

Reply:
My vote would be to keep this inside of the `repoServer` structure since it only corresponds to that service and recreate it as just as a single boolean value. I can update the PR if this is the direction you want to go towards.

Reply:
> My vote would be to keep this inside of the `repoServer` structure since it only correspo

Sure 👍 the only thing is that we don't preffer a subkey `repoServer.bla.enabled` if there is only an enabled/disabled option below the `bla` subkey. Subkey is fine if there are multiple options down there.

Reply:
Okay I updated it based on a few of the above comments and turned it into a single boolean under the `repoServer` section.